docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Computing/index.rst

```diff
@@ -146,7 +146,9 @@ Options for the HTCondorCEs
 | **Name**            | **Description**                                     | **Example**                                               |
 +---------------------+-----------------------------------------------------+-----------------------------------------------------------+
 | ExtraSubmitString   | Additional string for the condor submit             | request_cpus = 8 \\n periodic_remove = ...                |
-|                     | file. Separate entries with "\\n".                  |                                                           |
+|                     | file. Separate entries with "\\n".                  | To use actual proxy lifetime instead of server default:   |
+|                     |                                                     | delegate_job_GSI_credentials_lifetime = 0                 |
+|                     |                                                     | (HTCondor currently doesn't support proxy renewal)        |
```

Comment:
HTCondor does support proxy renewal when using a local schedd (at least).

Reply:
Hopefully now changed to make it clear that this is needed with remote scheduler (and there it really is).